types/braintree/package.json

```diff
@@ -23,6 +23,10 @@
         {
             "name": "Sanders DeNardi",
             "githubUsername": "sedenardi"
+        },
+        {
+            "name": "Sebastian Campos",
+            "githubUsername": "sebacampos"
```

Comment:
Are you doing this intentionally? This is the list of package owners, not a list of people who contributed.

Reply:
Makes sense to me given that the existing owners have not reviewed the code at all.